easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -12,18 +12,27 @@ sources = [SOURCE_TAR_GZ]
 source_urls = [GNU_SOURCE]
 
 # zlib required, but being linked instatically, so not a runtime dep
-builddependencies = [('zlib', '1.2.8')]
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.4'),
+    ('zlib', '1.2.8'),
+]
 
 # statically link with zlib, to avoid runtime dependency on zlib
 preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 
-# make sure that system libraries are also considered by ld
-configopts = '--with-sysroot=/'
+# make sure that system libraries are also considered by ld and ld.gold is also built
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default '
 
-binlist = ['addr2line', 'ar', 'as', 'c++filt', 'elfedit', 'gprof', 'ld', 'ld.bfd', 'nm',
-           'objcopy', 'objdump', 'ranlib', 'readelf', 'size', 'strings', 'strip' ]
+# If you require libiberty uncomment the following (you probably want a PIC build for linking purposes if you need libiberty):
```

Comment:
Reluctantly yes, libiberty used to be provided by binutils (up to 2.22 I think) though the real source is in GCC. The problem with that is it becomes a toolchain dependent library then. Our site customisation will leave this in I think.